### PR TITLE
Slide marquee take the text on the right from the gradient

### PR DESCRIPTION
**Pull request by DohyungLim**
- Issue created from the marquee slide effect.
- The last text on the right is obscured by the gradient.
- Get the text by raising the value of translateX.

### CODE REVIEW

File: src/js/core/widget/core/Marquee.js [hunk 1]
```diff
@@ -234,7 +234,7 @@
 						value,
 						returnValue;
 
-					value = state * (textWidth - containerWidth);
+					value = state * (textWidth - containerWidth + 25);
```
Comment (grzegorz-czajkowski) [inline src/js/core/widget/core/Marquee.js]: I've applied your PR and rebuild TAU. With your change I can see last character of text on right side (no cutting) in UI Components app. 
Please have in mind that this change is made to core (src/js/core/widget/core/Marquee.js) implementation and will applied for both Mobile and Wearable profiles. Such hard coded values may bring unexpected result in some cases. Can we find alternative way to fix it? For example by applying percentage etc.

**Review by grzegorz-czajkowski — CHANGES_REQUESTED** [review]

**Comment by grzegorz-czajkowski** [comment]
Don't we have the same issue for alternate style?

**Comment by singa2000** [comment]
A little issue also occurs in alternate style.

File: src/js/core/widget/core/Marquee.js (hunk 1 above)
Comment (DohyungLim) [reply src/js/core/widget/core/Marquee.js]: I measured the right gradient size at 15%.
and the size was divided by two and divided evenly.
I tested it on the Marquee Sample, the Snap Marquee List, and the Marquee List, and they all confirmed normal operation.

please re-check and report me.

**Review by TomaszLukawskiSam — APPROVED** [review]
I verified. I don't find any issues. Looks good to me.

**Review by singa2000 — COMMENTED** [review]
It's good working for me.

**Comment by singa2000** [comment]
@grzegorz-czajkowski 
Would you review the PR,please?

**Review by grzegorz-czajkowski — APPROVED** [review]
Thanks for taking my comment into considerations. Looks good to me.

**Pull request merged**